learn/tanssi/external-security-providers/symbiotic.md

```diff
@@ -110,20 +112,133 @@ flowchart LR
+
+```mermaid
+sequenceDiagram
+ participant Operator
+ participant OperatorRewardsContract
+ participant StakerRewardsContract
+ participant Stakers
+
+ Operator->>OperatorRewardsContract: 1. Claim rewards (Merkle proof)
+ OperatorRewardsContract->>Operator: 2. Release rewards to the operator
+ OperatorRewardsContract->>StakerRewardsContract: 3. Forward the remainder to StakerRewards
+ Stakers->>StakerRewardsContract: 4. Claim individual rewards
+```
+
+### Slashing {: #slashing }
+
+The Tanssi protocol implements slashing to penalize operators for misbehavior. When a slashing event is triggered, the authorities designated as resolvers by the vault managers can either accept or revert this action.
+
+The following actions can trigger slashing events:
+
+1. Producing Invalid Blocks (e.g., blocks that include invalid transactions)
+2. Invalid Validation (e.g., double-signing or breaking protocol rules)
 3. Downtime or Unavailability
 4. Consensus Violations
 
-When a veto-slashing event is triggered, the authorities designated as resolvers by the vault managers can accept or revert this action.
+!!!note
+    Slashing events can only be triggered by operators' misbehavior within the Tanssi Network. Even if Tanssi networks are faulty or malicious, they operate in a sandboxed environment and cannot cause slashing.
 
-!!! note
-    Slashing events can only be triggered by operators' misbehavings in the Tanssi Network itself. Tanssi networks, even if faulty or malicious, run in a sandboxed environment and can not cause slashing.
\ No newline at end of file
+#### **Slashing Process** {: #slashing-process }
+
+The slashing process follows a path similar to that of rewards. When a validator misbehaves, the Tanssi Network sends a slashing request message to the trustless bridge (Snowbridge). This process resembles the reward, where the message passes through the gateway and into the `Middleware` where the slashing method gets called.
+
+The slashing method receives a unique identifier for the operator's identity, the severity of the slash as a percentage of the operator's assigned stake in each vault, and the time context within which the offense occurred.
+
+The slashing process consists of the following steps:
+
+1. Slash Request - Tanssi sends the slash request to the `Middleware` with the parameters `operatorKey`, percentage, and epoch
+2. Operator Validation - the `Middleware` validates the operator's identity and checks if the operator is eligible for slashing
+3. Vault Iteration - the `Middleware` iterates through all active vaults during the offense epoch, skipping any inactive vaults
+4. Retrieve Operator Stake - for each active vault, the Middleware retrieves the stake of the misbehaving operator
+5. Calculate Slash Amount - the `Middleware` calculates the slashing amount by applying the slashed percentage to the operator's stake in each vault
+6. Slashing - Depending on the vault's slashing implementation, there are two possible routes
```

Comment:
Same comments on the list

Reply:
Also watch capitalization of the first word in the description (to the right of the hypen) - the last item in this list uses capitalized "Depending"